README.md

```diff
@@ -12,3 +12,7 @@ This increases the chance that we might be able to use your contribution (or it
 
 Note that all commits should be signed using a gpg key.
 
+## Documentation
+All documantation can be found in the `architecture` and `docs` directories.
```

Comment:
Thanks! Useful contribution! Small typo though: documantation -> documentation.

Reply:
Fixed!

Feel free to squash when merging.